llvm/lib/Target/DirectX/DXILOpBuilder.cpp

```diff
@@ -289,24 +289,25 @@ static ShaderKind getShaderKindEnum(Triple::EnvironmentType EnvType) {
 ///               its specification in DXIL.td.
 /// \param OverloadTy Return type to be used to construct DXIL function type.
```

Comment:
The doxygen comment about getDxilOpFunctionType is now out of date.

I'm not sure that the list of parameters in the comment is really adding much, but I think the mismatch would generate a Doxygen warning so it'd be worth resolving this one way or another.

Reply:
Removed the parameters from the comment